docs/DeveloperGuide.md

```diff
@@ -283,32 +283,155 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `TrackO` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC01 - Add an order**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. User requests to add an order.
+2. System adds the order.
+
+   Use case ends.
+
+**Extensions**
+
+- 1a. User inputs incomplete order data. <br>
+  - 1ai. System informs user of the incomplete data.
+  
+    Use case ends.
+
+**Use case: UC02 - Delete an order**
+
+**MSS**
+
+1. User requests to delete an order.
+2. System deletes the order.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+- 1a. The order the user wishes to delete does not exist.
+      
+  - 1ai. System informs the user of the non-existent order.
+    
+     Use case ends.
+- 2a. The list has no orders.
+  
+  - 2ai. System informs the user of the empty order list.
+  
+    Use case ends.
 
-  Use case ends.
+**Use case: UC03 - Listing orders**
 
-* 3a. The given index is invalid.
+**MSS**
 
-    * 3a1. AddressBook shows an error message.
+1. User requests to list all orders.
+2. System lists all orders.
 
-      Use case resumes at step 2.
+    Use case ends.
 
-*{More to be added}*
+**Extensions**
+
+- 2a. The list of orders is empty.
+    - 2ai. System informs the user that the list is empty.
```

Comment:
I could be mistaken, but I think the numbering here should be **2a** then **2a1**, i.e, no use of roman numerals for sublisting.

Do check all the other extensions as well!

Reply:
Ah, it's my bad! Fixed all of them :D thank you for pointing it out!